### ISSUE TITLE
Retrieve logs from pod

### ISSUE THREAD

**Issue by kfucGG** [fabric8io/kubernetes-client#5395]
Hello guys, i have spring boot application which running in openshift pod, is openshift-client have some api for retrieve Sys.out from this pod?

**Comment by rohanKanojia** [comment]
Yes, please take a look at [PodLogExample](https://github.com/fabric8io/kubernetes-client/blob/main/kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PodLogExample.java#L47)

**Comment by kfucGG** [comment]
Thx
